frontend/src/graphql/Mutation/ObjectiveLink.js

```diff
@@ -0,0 +1,56 @@
+import gql from 'graphql-tag'
+
+const CREATE_OBJECTIVE_LINK = gql`
+mutation createObjectiveLink($objectiveId: ID!, $courseId: ID!, $workspaceId: ID!) {
+    __typename
```

Comment:
This seems to be missing the actual mutation call and should probably include text and weight as parameters